tests/check/test_stream.c

```diff
@@ -192,6 +199,11 @@ test_stream_item_create_from_type(char type, uint8_t id, SignedVideoCodec codec)
       bu_data = codec == SV_CODEC_H264 ? p_nalu_h264
                                        : (codec == SV_CODEC_H265 ? p_nalu_h265 : invalid_av1);
       break;
+    case 'O':
+      bu_data = codec == SV_CODEC_H264 ? oms_sei_nalu_h264
+                                       : (codec == SV_CODEC_H265 ? oms_sei_nalu_h265 : NULL);
+      bu_data_size = (codec != SV_CODEC_AV1) ? DUMMY_SEI_SIZE : 0;
```

Comment:
This will still create an item further down, but without valid content. Also, even if you pass in zero as size in memcpy I think the source parameter needs to be valid, that is, not NULL.

I suggest that you handle this such that this function returns no item if `bu_data = NULL`.

Look at `test_stream_create()` and how items are treated.